tests/llsd_test.py

```diff
@@ -1503,6 +1503,24 @@ def testFormatPrettyXML(self):
 
         output_xml = llsd.format_pretty_xml(python_object)
 
+        with open("foo.llsd.xml", "wb") as f:
+            f.write(output_xml)
+
+        self.assertEqual(output_xml.decode("utf8"), """<?xml version="1.0" ?>
+<llsd>
+<map>
```

Comment:
This matches prior versions of LLSD: the starting element after `<llsd>` is not indented.